src/main/kotlin/com/google/androidstudiopoet/BuildGradle.kt

```diff
@@ -63,5 +63,6 @@ object BuildGradle {
                 "    }\n" +
                 "}\n"
             else
-                ""
+                "" ) +
+            (extraLines?.joinToString(separator = "\n") ?: "")
```

Comment:
Can you please extract `?.joinToString(separator = "\n") ?: ""` to a separate extention function?

Reply:
Created a separate extension, it can be applied to a Collection\<Any\>?